### PR TITLE
les-9 Pages Widgets

### PR DESCRIPTION
**Pull request by Ihorzolotar**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/ZolotarModule/CustomWidget/Block/Widget/HomepageBanner.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PalashAs\CustomWidget\Block\Widget;
```
Comment (chabanenk0) [inline app/code/ZolotarModule/CustomWidget/Block/Widget/HomepageBanner.php]: PalashAs is a name of different student. Please correct, seems it shouldn't work

File: app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php if ($block->getBlockId()): ?>
```
Comment (chabanenk0) [inline app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml]: Please move this code fragment to your block  (together with checking if block exists to your block 
ZolotarModule\CustomWidget\Block\Widget\HomepageBanner
and call that method here. For example:
 echo $block->getConnectedBlockHtml()

Comment (Ihorzolotar) [reply app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml]: I did not quite understand what needs to be changed, and for what. Please explain.

Comment (APanteleychuk) [reply app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml]: You need to create one method in block class with check if exist block and rendering functionality

File: app/code/ZolotarModule/CustomWidget/Block/Widget/HomepageBanner.php (hunk 1 above)
Comment (Ihorzolotar) [reply app/code/ZolotarModule/CustomWidget/Block/Widget/HomepageBanner.php]: done

File: app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?php if ($block->getTitle()): ?>
+    <a class="action button"
+            href="geekhub-cms">
+        <span><?php echo $block->getTitle(); ?></span>
```
Comment (chabanenk0) [inline app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml]: Block id should be used somewhere (block loading, rendering). Please add this procedure in your block and render the block content here

Comment (Ihorzolotar) [reply app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml]: It is added by the database. I'll fix it.

File: app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml (hunk 2 above)
Comment (Ihorzolotar) [reply app/code/ZolotarModule/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml]: I don’t quite understand how to transfer the code of the created function from the file app/code/ZolotarModule/CustomWidget/Block/Widget/HomepageBanner.php to the template in order for the condition to work.

**Pull request merged**